### PR TITLE
Scrim

### PR DESCRIPTION
**Pull request by wp-aberg**
This PR creates a new Scrim component that can be re-used across other components

### CODE REVIEW

File: ui/drawer/src/DrawerContent.tsx [hunk 1]
```diff
@@ -139,35 +144,57 @@ export const DrawerContent = React.forwardRef<
...
       innerClassName,
+      loopFocus = true,
+      trapFocus = false,
       ...props
     },
     ref
   ) => {
     const context = React.useContext(DrawerContext);
 
+    function handleEnter() {
```
Comment (atomictangerine) [inline ui/drawer/src/DrawerContent.tsx:155]: 100% stylistic, but I noticed that we're not being consistent with using the `function` vs the es6 arrow functions. We might want to choose one as a team?

File: ui/drawer/src/play.stories.jsx [hunk 2]
```diff
@@ -46,6 +46,46 @@ Drawer.parameters = {
   chromatic: { disableSnapshot: true },
 };
 
+const FocusTemplate = ({ loopFocus, ...args }) => {
+  return (
+    <>
```
Comment (atomictangerine) [inline ui/drawer/src/play.stories.jsx:51]: you can remove this react fragment. You can also remove the return and just use the `() => ()` default return (I forget what its official name is)

Comment (atomictangerine) [reply ui/drawer/src/play.stories.jsx:51]: But not too important cause it's for storybook

**Review by atomictangerine — APPROVED** [review]

File: ui/scrim/jest-setup.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+import "@testing-library/jest-dom";
```
Comment (artmsilva) [inline ui/scrim/jest-setup.ts:1]: Jest is set up at the root. I think it's fine to move all (including Drawer) of the Jest configs up there.

Comment (artmsilva) [reply ui/scrim/jest-setup.ts:1]: https://github.com/washingtonpost/wpds-ui-kit/pull/211/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R172

**Pull request merged**